### DISCUSSION TITLE
GitHub Desktop - I've screwed up!

### DISCUSSION THREAD

**Discussion by ItisNiGel** [community/community#22594] [New to GitHub]
<p>Hi,</p>
<p>My first venture into the dark art of Git!</p>
<p>I thought I would start by dipping my toe into GitHub Desktop first! That hasn’t gone well!</p>
<p>Looks like I’ve added my C drive! WT! 72000 of files. I’ve tried to remove them but then this just freezes the application!</p>
<p>How can I delete/remove this from the app PLEASE?</p>
<p>Regards,</p>
<p>Nigel ;-/</p>
<p>  </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/70cdcd6361bf71a8481368907b41d608e53e5410.jpeg" data-download-href="/uploads/short-url/g5UpaY84OpBOnhO6b0vW8Q1T38Y.jpg?dl=1" title="git_desktop.JPG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10099150/181072104-8b909a74-9774-45dd-b7fe-bfbd2d5af2fb.jpg" alt="git_desktop.JPG" data-base62-sha1="g5UpaY84OpBOnhO6b0vW8Q1T38Y" width="690" height="479" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/70cdcd6361bf71a8481368907b41d608e53e5410_2_690x479.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/70cdcd6361bf71a8481368907b41d608e53e5410_2_1035x718.jpg 1.5x, https://aws1.discourse-cdn.com/github/original/1X/70cdcd6361bf71a8481368907b41d608e53e5410.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/70cdcd6361bf71a8481368907b41d608e53e5410_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">git_desktop.JPG</span><span class="informations">1257×874 104 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/87e3dc3b398e55a908e3b5b397ee48293906c276.jpeg" data-download-href="/uploads/short-url/jo8E6HRSwNDBLRocCgngRt9uLHg.jpg?dl=1" title="git_desktop_2.JPG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10099150/181072106-2ddac6ee-8102-4297-928e-fcb625f1852c.jpg" alt="git_desktop_2.JPG" data-base62-sha1="jo8E6HRSwNDBLRocCgngRt9uLHg" width="599" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/87e3dc3b398e55a908e3b5b397ee48293906c276_2_599x500.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/87e3dc3b398e55a908e3b5b397ee48293906c276_2_898x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/original/1X/87e3dc3b398e55a908e3b5b397ee48293906c276.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/87e3dc3b398e55a908e3b5b397ee48293906c276_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">git_desktop_2.JPG</span><span class="informations">1188×991 103 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by AndreaGriffiths11** [comment]
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our community organized so users can more easily find information.</p>
<p>As you’ll notice, your topic is now here in the How to Use Git and GitHub. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>
<p>Andrea</p>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/iamnigelfrancis">@iamnigelfrancis</a> here’s how you can fix this:</p>
<ol>
<li>Right-click on the repository in the repository list in GitHub Desktop and select <code>Remove</code> to remove the repository</li>
<li>Open File Explorer and go to your C:\ drive</li>
<li>Make sure you have the option to <a href="https://support.microsoft.com/en-us/help/14201/windows-show-hidden-files" rel="nofollow noopener">view hidden files and folders</a> enabled</li>
<li>Delete the hidden <code>.git</code> directory located there</li>
</ol>
<p>These steps will remove the repository from GitHub Desktop and delete the repository, but will not delete any of your existing files on the C:\ drive. Let me know if you run into any issues with these steps!</p>

**Reply by EduCoimbra** [reply]
Thank you for your help!

**Reply by Vshah141** [reply]
What i did is that i have added an existing repository from github desktop and added my folder path having multiple files and then after some time i removed that repo from my github desktop now what shall i do i am unable to see the files in my pc or in my github

**Comment by ItisNiGel** [comment]
<p>Andrea,</p>
<p>Many thanks!</p>
<p>If my initial post was on Stackoverflow and I’d written that, I’d have people breathing fire and throwing spears at me! A sole reason I don’t use SOF!</p>
<p>I may be using Use Git and GitHub quite a bit!</p>
<p>Again, thank you for your kind direction and no spear throwing! </p>
<p>Nigel 😀</p>

**Comment by ItisNiGel** [comment]
<p>Steve,</p>
<p>Big hugs!</p>
<p>Thank you 😀</p>
